portfolio/src/main/webapp/leave-comment/script.js

```diff
@@ -0,0 +1,37 @@
+function getComments() {
+  console.log("Downloading comments.");
+  const responsePromise = fetch("/get-comments");
```

Comment:
it is unusual to move each promise step in a variable. I recommend to use a [fluid interface style](https://de.wikipedia.org/wiki/Fluent_Interface) here, i.e. 
```javascript
fetch("/get-comments")
  .then(response => response.text)
  .then(JSON.parse)
  ....
```

Reply:
Done.